example/k3d/lib/tempo/main.libsonnet

```diff
@@ -0,0 +1,79 @@
+local k = import 'ksonnet-util/kausal.libsonnet';
+
+local configMap = k.core.v1.configMap;
+local container = k.core.v1.container;
+local containerPort = k.core.v1.containerPort;
+local deployment = k.apps.v1.deployment;
+local pvc = k.core.v1.persistentVolumeClaim;
+local service = k.core.v1.service;
+local volumeMount = k.core.v1.volumeMount;
+local volume = k.core.v1.volume;
+
+{
```

Comment:
This is pretty useful. Do you think it would make sense to make this part of jsonnet-libs or something similar to that we can share the Tempo setup?

Reply:
I don't think we should share this; tempo [already has its own jsonnet library](https://github.com/grafana/tempo/tree/master/operations/jsonnet) but I built this to quickly hack together a single-replicaed service (there's similar files for Loki and Cortex). 

It'd probably be better to use the single-process library Joe wrote rather than having the extra boilerplate here, though single-process libraries don't exist for Loki/Cortex.